tools/azure-pipelines/build-apache-repo.yml

```diff
@@ -103,9 +103,9 @@ stages:
       - template: build-nightly-dist.yml
         parameters:
           stage_name: cron_snapshot_deployment
-          environment: PROFILE="-Djdk17 -Pjava17-target"
+          environment: PROFILE="-Djdk11 -Pjava11-target"
```

Comment:
Can we put a comment somewhere before that line why we change it back and what should be checked/done in order to move again to jdk17?

Reply:
👍 Make sense, updated.